lighthouse-core/test/report/html/renderer/report-ui-features-test.js

```diff
@@ -191,7 +191,8 @@ describe('ReportUIFeatures', () => {
 
         function getUrlsInTable() {
```

Comment:
can we get a test for the subitem filtering in sync with their parent row? :)

Reply:
done